src/pyjion/absvalue.h

```diff
@@ -459,8 +466,10 @@ extern SliceValue Slice;
 extern ComplexValue Complex;
 extern BuiltinValue Builtin;
 extern IterableValue Iterable;
+extern ModuleValue Module;
 
-AbstractValue* avkToAbstractValue(AbstractValueKind);
+AbstractValue *avkToAbstractValue(AbstractValueKind){
```

Comment:
This line should remain as it was before. Header files (`.h`) should only have the function signatures, not the implementations.